diff --git a/dist/content/utils.js b/dist/content/utils.js
index a3ed65a87d9f3c15d550ad03752e615446b6fe7a..61dd3ad18c1d308cebc47ba3d2e0c31008ffa9ef 100644
--- a/dist/content/utils.js
+++ b/dist/content/utils.js
@@ -76,7 +76,7 @@ const collectionConfigParser = z.union([
             return z.NEVER;
           }
         }).optional(),
-        createSchema: z.function({
+        createSchema: z.optional(z.function({
           input: [],
           output: z.promise(
             z.object({
@@ -84,7 +84,7 @@ const collectionConfigParser = z.union([
               types: z.string()
             })
           )
-        }).optional()
+        }))
       })
     ])
   }),
@@ -640,7 +640,7 @@ function contentModuleToId(fileName) {
   return `${DEFERRED_MODULE}?${params.toString()}`;
 }
 function safeStringifyReplacer(seen) {
-  return function(_key, value) {
+  return function (_key, value) {
     if (!(value !== null && typeof value === "object")) {
       return value;
     }
